GannettDigital/dmpt-candidate-exercise
# New Feature : Inline Advertisements

# Overview
This feature adds inline advertisement units (300x250s) with the article body. 

# Requirements
1. Inline ads must request/render only 300x250's (see <https://developers.google.com/doubleclick-gpt/reference#googletag.defineSlot> for gpt documentation)
2. First ad must be at least 200 words from the top of the article
3. Each subsequent ad must be at least 100 words from the last inline ad
4. Ads must not be adjacent to `<aside>` or `<figure>` elements
5. Ads must be between two paragraphs. The only exception is when an available spot precedes a `<h2>`
6. Inline ads must lazy load when they come into view
7. Inline ads must be centered and have a label above them that reads "Advertisement - Read More"
8. Must work on IE 11

# To Test
1. Pull down branch
2. Open index.html in browser of choice
3. Inline ad units should appear between paragraphs and render lazily
